src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
implement main according to the task description

Reply:
fix comment
if you have questions ask in chat
Not main method is completely wrong